chapter1/element.md

```diff
@@ -32,24 +32,24 @@ def_options interval: [
 
 ## Pads
 
-It is very common for the filter to declare two pads that are called :source and :sink
+It is very common for the filter to declare two pads that are called :input and :output
 
 ```elixir
-def_known_source_pads source: {:always, :pull, :any}
-def_known_sink_pads sink: {:always, {:pull, demand_in: :bytes}, :any}
+def_input_pads input: [availability: :always, mode: :pull, demand_unit: :bytes, caps: :any]
+def_output_pads output: [availability: :always, mode: :pull, caps: :any]
 ```
 
-In above definition, atom :always means that pad of this element is always available. The other option is `:on_request` which means that pad is being created on request, for example, during the 'playing' state.
+In above definition, availability :always means that pad of this element is always available. The other option is `:on_request` which means that pad is being created on request, for example, during the 'playing' state.
 
-:pull declares the mode of the pad. It means that this element sends buffers to the next element only when they are demanded. Demands are received in `handle_demand` callback and they may refer to the number of bytes or buffers to send. The unit is specified by next argument - `demand_in: :bytes`.
+:pull mode means that this element sends buffers to the next element only when they are demanded. Demands are received in `handle_demand` callback and they may refer to the number of bytes or buffers to send. The unit is specified by next argument - `demand_in: :bytes`.
 
 The other option is :push mode, that means that element will send buffers whenever it wants or whenever they are available. In this case, specifying demand unit is unnecessary.
 
-The third element in the tuple represents the capabilities of the pad. `:any` means that every type of buffers can be passed on this pad.
+The next element in the keyword list represents the capabilities of the pad. `:any` means that any type of buffer can be passed on this pad.
```

Comment:
I believe the caps deserve a bit more explanation i.e. there should be some info about what else can be passed here